test/unit/presenters/publishing_api/publication_presenter_test.rb

```diff
@@ -154,27 +154,6 @@ def present(edition)
     assert_equal [location.content_id], presented_item.content[:links][:world_locations]
   end
 
-  test "documents include the alternative format contact email with a direct email link" do
-    publication = create(:publication, :with_command_paper)
-    GovukPublishingComponents::Presenters::AttachmentHelper.stub_const(:EMAILS_IN_ACCESSIBLE_FORMAT_REQUEST_PILOT, []) do
-      presented_item = present(publication)
-      document = presented_item.content[:details][:documents].first
-      assert document.include?("This file may not be suitable for users of assistive technology.")
-      assert document.include?("mailto:#{publication.alternative_format_provider.alternative_format_contact_email}")
-    end
-  end
```

Comment:
We probably want to keep the premise of this test, probably just a copy / paste from the previous version: https://github.com/alphagov/whitehall/blame/86bbd48cfc106ff05960b407fa04f6d7bb234338/test/unit/presenters/publishing_api/publication_presenter_test.rb#L153-L159

Reply:
Ah yes, I think I got confused looking at the original PR. Thanks! Reinstated.

Reply:
Awesome, cheers!